src/src/DataStructs/SettingsStruct.h

```diff
@@ -279,7 +279,10 @@ class SettingsStruct_tmpl
   int16_t       TaskDevicePluginConfig[N_TASKS][PLUGIN_CONFIGVAR_MAX];
   boolean       TaskDevicePin1Inversed[N_TASKS] = {0};
   float         TaskDevicePluginConfigFloat[N_TASKS][PLUGIN_CONFIGFLOATVAR_MAX];
-  long          TaskDevicePluginConfigLong[N_TASKS][PLUGIN_CONFIGLONGVAR_MAX];
+  union {
+    long     TaskDevicePluginConfigLong[N_TASKS][PLUGIN_CONFIGLONGVAR_MAX];
```

Comment:
Please use `int32_t` instead of long, just to be sure.

Reply:
Cherry picked from other branch